pyro/params/param_store.py

```diff
@@ -4,10 +4,13 @@
 import re
 import warnings
 import weakref
+from collections.abc import Iterator, KeysView
```

Comment:
@willtai I think you need to replace `collections.abc.Iterator` with `typing.Iterable` for tests to pass

Reply:
thanks! However, I imported `typing.Iterator` rather than `typing.Iterable`